test/jdk/java/foreign/nested/TestNested.java

```diff
@@ -175,10 +181,10 @@ public static Object[][] nestedLayouts() {
     ).withName("S8");
     static final StructLayout S9 = MemoryLayout.structLayout(
             C_CHAR.withName("f0"),
-            MemoryLayout.paddingLayout(7),
+            MemoryLayout.paddingLayout(isAixOS ? 3 : 7),
```

Comment:
As [I said](https://github.com/eclipse-openj9/openj9/issues/18287#issuecomment-1766801519), I believe this test is fundamentally broken in that it assumes all platforms pad the same amount.

In any case, I suggest this need not be platform-specific; we can instead say
```java
            MemoryLayout.paddingLayout(C_DOUBLE.byteAlignment() - 1),
```
here and on line 187.

Reply:
Agreed and updated against the suggestion above.